packages/node-app/src/app.ts

```diff
@@ -1,14 +1,17 @@
-import express from "express";
-import { router as GiveawayRoutes } from "./routes/giveaway.route";
-import { router as LocationRoutes } from "./routes/location.routes";
+import cors from 'cors';
+import express from 'express';
+
+import { router as GiveawayRoutes } from './routes/giveaway.route';
+import { router as LocationRoutes } from './routes/location.routes';
 
 export const app = express();
 
+app.use(cors({ origin: ['http://localhost:4200'] }));
```

Comment:
Let's use an `env` variable for this origin since it will have a different value depending on the environment. You can add the development value to the `.env.example` to make the setup quicker